book/website/communication/citable/citable-linking.md

```diff
@@ -0,0 +1,64 @@
+(cm-citable-linking)=
+
+# Linking Research Objects
+
+Linking your data, code and article will ensure that you receive **credit** for your work and it will increase the visibility of your research outputs.
+There are several different ways to do this. 
+You can decide to 1) open up your data publicly, at the same time as for example the **preprint** of your article, or 2) keep the dataset closed until the article is published.
```

Comment:
I _may_ be misreading this, but I think it works without the "for example". :)

Suggestion: split this sentence into an ordered list:

> You can decide to:
> 1. open up your data publicly, at the same time as ~~for example~~ the **preprint** of your article, or 
> 2. keep the dataset closed until the article is published.

Reply:
Thanks! I'll split it into an ordered list indeed. As not everyone publishes preprints I'll keep the for example in there though.